### PR TITLE
[stable/node-problem-detector] adding 3 new maintainers

### PR DESCRIPTION
**Pull request by max-rocket-internet**
#### What this PR does / why we need it:

Add 3 maintainers for https://github.com/helm/charts/issues/13867

Is that OK?

@Random-Liu
@andyxning
@wangzhen127

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

**Comment by max-rocket-internet** [comment]
FYI @maorfr

File: stable/node-problem-detector/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: node-problem-detector
-version: "1.4.1"
+version: "1.42"
```
Comment (andyxning) [inline stable/node-problem-detector/Chart.yaml]: Seems this should be `1.4.2` instead of `1.42`.

Comment (max-rocket-internet) [reply stable/node-problem-detector/Chart.yaml]: Ah yes, typo in my rebase. Thanks.

**Comment by maorfr** [comment]
/assign
/ok-to-test

please add the new maintainers to Chart.yml as well :)

**Comment by max-rocket-internet** [comment]
@maorfr 

> please add the new maintainers to Chart.yml as well :)

Done!

**Comment by maorfr** [comment]
/lgtm

thanks!

**Comment by Random-Liu** [comment]
Tx for doing this!

**Comment by maorfr** [comment]
Thank you for getting involved!

**Pull request merged**